modules/gapi/src/backends/common/serialization.hpp

```diff
@@ -111,6 +113,16 @@ GAPI_EXPORTS I::IStream& operator>> (I::IStream& is,       cv::gimpl::Protocol &
 GAPI_EXPORTS I::OStream& operator<< (I::OStream& os, const cv::GArg &arg);
 GAPI_EXPORTS I::IStream& operator>> (I::IStream& is,       cv::GArg &arg);
 
+GAPI_EXPORTS I::OStream& operator<< (I::OStream& os, const cv::GMetaArg &arg);
+GAPI_EXPORTS I::IStream& operator>> (I::IStream& is,       cv::GMetaArg &arg);
+
+GAPI_EXPORTS I::OStream& operator<< (I::OStream& os, const cv::GRunArg &arg);
+GAPI_EXPORTS I::IStream& operator>> (I::IStream& is, cv::GRunArg &arg);
+
+GAPI_EXPORTS I::IStream& operator >> (I::IStream& is, cv::GRunArgP &arg);
+
+GAPI_EXPORTS void bind(cv::GRunArg &res_obj, cv::GRunArgP &out_obj);
```

Comment:
The right place to define `bind` is the public header where we define `GRunArgs` and `GRunArgsP`.

Also, the idea of the bind is to work on `GRunArgs` (a vector) but not a single `GRunArg`.

Reply:
Done